aeron-driver/src/test/c/aeron_system_test.cpp

```diff
@@ -42,15 +45,39 @@ class SystemTest : public testing::Test
 
 TEST_F(SystemTest, shouldReclaimSubscriptionWhenOutOfScopeAndNotFound)
```

Comment:
This test was testing something specific to the C++ API involved with NOT doing a find on the Subscription. This test does not act that way and C API does not work the same. So, it might be better to just remove the test at this point.

Reply:
Okay, makes sense.  I'll move it over to the C++ Tests.

Reply:
I've moved the test across.  This has meant duplication of the `EmbededMediaDriver.h`.